composeApp/src/commonMain/kotlin/org/ooni/probe/data/repositories/PreferenceRepository.kt

```diff
@@ -147,7 +147,7 @@ class PreferenceRepository(
         isAutoRun: Boolean,
     ): Flow<Boolean> =
         dataStore.data.map {
-            it[booleanPreferencesKey(getNetTestKey(descriptor, netTest, isAutoRun))] == true
+            it[booleanPreferencesKey(getNetTestKey(descriptor, netTest, isAutoRun))] ?: netTest.defaultPreferenceValue(isAutoRun)
```

Comment:
If we're already bootstrapping the preferences correctly, why do we the default here? That's a different approach from the other defaults we have. I feel we can skip this, and just have them set on `BootstrapPreferences`, but maybe I'm missing something.

Reply:
We bootstrap references in multiplatform but the android app has default preferences that are not available on disk untill the user changes them.